src/software/thunderscope/play/playinfo_widget.py

```diff
@@ -103,5 +114,12 @@ def refresh(self):
             }
         )
 
-        self.resizeColumnsToContents()
-        self.resizeRowsToContents()
+        self.play_table.resizeColumnsToContents()
+        self.play_table.resizeRowsToContents()
+        if referee_msg_dict:
```

Comment:
Can you tag #2412 here as well. There are more information which we can add here, and we can probably improve the layout of the info.